development/yabasic/slack-desc

```diff
@@ -16,4 +16,4 @@ yabasic: Yabasic runs under Unix and Windows and has a comprehensive
 yabasic: documentation; it is small, simple, open-source and free.
 yabasic:
 yabasic: Yabasic is very well documented.
-yabasic:
+yabasic: Homepage at http://2484.de/yabasic/
```

Comment:
please don't put HOMEPAGE in README

Reply:
I actually used the URL in the slack-desc as part of my workflow, but ok, I'll arrange something different...